### PR TITLE
PE-777, PE-712, PE-792, Change buttons color and fix logo url.

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description 
Change buttons backgroung and fix logo url.

### CODE REVIEW

File: edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss [hunk 1]
```diff
@@ -1741,3 +1741,13 @@ button {
     color: #3764f0;
   }
 }
+
+.btn.login-provider {
+  background: #126f9a;
+  border: 5px solid #126f9a;
+}
+
+.recap_actions .btn{
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss]: Space before open curly brace. @andrey-canon

File: edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss [hunk 2]
```diff
@@ -1741,3 +1741,13 @@ button {
     color: #3764f0;
   }
 }
+
+.btn.login-provider {
+  background: #126f9a;
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss]: Is there a scss variable for: #126f9a? @andrey-canon

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 3]
```diff
@@ -1,12 +1,21 @@
...
+{% load static %}
 
 {% get_current_language as LANGUAGE_CODE %}
 {% get_current_language_bidi as LANGUAGE_BIDI %}
 
 {# This is preview text that is visible in the inbox view of many email clients but not visible in the actual #}
 {# email itself. #}
 
+<style type="text/css">
+
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: No space required. @andrey-canon

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 4]
```diff
@@ -1,12 +1,21 @@
...
+    .logo-image {
+        width: 225px;
+    }
+
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: No space required. @andrey-canon

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 5]
```diff
@@ -1,12 +1,21 @@
...
 {% get_current_language as LANGUAGE_CODE %}
 {% get_current_language_bidi as LANGUAGE_BIDI %}
 
 {# This is preview text that is visible in the inbox view of many email clients but not visible in the actual #}
 {# email itself. #}
 
+<style type="text/css">
+
+    .logo-image {
+        width: 225px;
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: Why 225px? Is there an special requirement? How does it look like in mobile views? @andrey-canon

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 6]
```diff
@@ -60,9 +70,8 @@
                 <table role="presentation" width="100%" align="left" border="0" cellpadding="0" cellspacing="0">
                     <tr>
                         <td width="70">
-                            <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://www.teachfirst.org.uk/profiles/charity/themes/ultimate/logo.svg" width="225"
-                                    height="40" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
+                            <a href="{% with_link_tracking homepage_url %}"><img class="logo-image"
+                                    src="{% with_link_tracking logo_url %}"  alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: Wrong indentation handling. @andrey-canon

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 5 above)
Comment (andrey-canon) [reply edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: That is the value that was in the inline styles

File: edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss (hunk 2 above)
Comment (andrey-canon) [reply edx-platform/teachfirst-theme/lms/static/sass/partials/lms/theme/_button.scss]: no

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 5 above)
Comment (Squirrel18) [reply edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: How does it look like in mobile views? @andrey-canon

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html [hunk 7]
```diff
@@ -60,9 +68,7 @@
                 <table role="presentation" width="100%" align="left" border="0" cellpadding="0" cellspacing="0">
                     <tr>
                         <td width="70">
-                            <a href="{% with_link_tracking homepage_url %}"><img
-                                    src="https://www.teachfirst.org.uk/profiles/charity/themes/ultimate/logo.svg" width="225"
-                                    height="40" alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
+                            <a href="{% with_link_tracking homepage_url %}"><img class="logo-image" src="{% with_link_tracking logo_url %}"  alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/></a>
```
Comment (Squirrel18) [inline edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]:
```suggestion
                            <a href="{% with_link_tracking homepage_url %}">
                                <img class="logo-image" src="{% with_link_tracking logo_url %}"  alt="{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}"/>
                            </a>
```

File: edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html (hunk 5 above)
Comment (andrey-canon) [reply edx-platform/teachfirst-theme/lms/templates/ace_common/edx_ace/common/base_body.html]: good

**Pull request merged**